lib/config.js

```diff
@@ -0,0 +1,3 @@
+var configPath = process.env.CONFIG_PATH || '../config';
```

Comment:
It's a little confusing to have 2 config.js files. lib/config.js requires ../config.js. Anyone loading the config file directly may accidentally miss this file, and it's a hard mistake to spot.

Reply:
Good point. I'll rename this one.